src/oc_erchef/habitat/plan.sh

```diff
@@ -24,7 +24,11 @@ pkg_upstream_url="https://github.com/chef/chef-server"
 
 pkg_exposes=(port)
 pkg_exports=(
-    [port]=oc_chef_wm.port
+  [port]=oc_chef_wm.port
+  [data_collector_enabled]=data_collector.enabled
+  [data_collector_server]=data_collector.server
+  [data_collector_port]=data_collector.port
+  [data_collector_token]=data_collector.token
```

Comment:
we need to figure how to just export the top level toml table key `[data_collector]=data_collector`

whenever I tried it would get:
```
oc_erchef_1          | hab-sup(MR): Starting jeremymv2/oc_erchef
oc_erchef_1          | thread 'main' panicked at 'Struct should serialize to bytes: ValueAfterTable', /checkout/src/libcore/result.rs:906:4
oc_erchef_1          | note: Run with `RUST_BACKTRACE=1` for a backtrace.
```